cleanup_utils/load_balancer_cleanup.py

```diff
@@ -47,8 +51,15 @@ def __init__(self, load_balancer_address, fragment_prefix, age_limit, dry_run):
 
     def run_cleanup(self):
         """Run the actual cleanup"""
+        logger.info("\n --- Starting Consul cleanup ---")
+        self._clean_consul()
```

Comment:
This should come after the self.dry_run` check below. 
No keys should be deleted when running with `dry_run`, but the log output should work as usual.

Reply:
I was just working on that and pushed some changes to it.